ibm/service/scc/data_source_ibm_scc_profile_attachment_test.go

```diff
@@ -13,12 +14,17 @@ import (
 )
 
 func TestAccIbmSccProfileAttachmentDataSourceBasic(t *testing.T) {
+	instanceID, ok := os.LookupEnv("IBMCLOUD_SCC_INSTANCE_ID")
```

Comment:
instead of duplication code in every testcase can you add a method in accTest.go file 
"TestAccPreCheckEnterpriseAccountImport" similar func